lib/taskana-core/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -59,6 +59,38 @@ void should_NotSetAttachmentSummariesToNull_When_CreatingNewTaskWithTaskService(
     assertThat(task.asSummary().getAttachmentSummaries()).isNotNull();
   }
 
+  @WithAccessId(user = "user-taskrouter")
+  @Test
+  void should_CreateTask_When_UserIsMemberOfTaskRouterRole() throws Exception {
+    TaskService taskService = taskanaEngine.getTaskService();
+    Task newTask = taskService.newTask("WBI:100000000000000000000000000000000006");
+    newTask.setClassificationKey("T2100");
+    ObjectReferenceImpl objectReference =
+        createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", "1234567");
+    objectReference.setTaskId(newTask.getId());
+    newTask.setPrimaryObjRef(objectReference);
+
+    Task createdTask = taskService.createTask(newTask);
+
+    assertThat(createdTask).isNotNull();
+  }
+
+  @WithAccessId(user = "user-1-1", groups = "cn=routers,cn=groups,OU=Test,O=TASKANA")
```

Comment:
user-1-1 is not authorized for the workbasket according to test-data/workbasket-access-list.sql.  The tests fails after removing the groups. cn=routers is mentioned in taskana.properties, so it can be used in tests. There is no .ldif file for taskana-core

Reply:
yes, you are right. The resolution of the groups happens far before that point...